### PR TITLE
Adsb 14862 replace chart bundles

### PR DESCRIPTION
**Pull request by nikeflight**

### CODE REVIEW

**Review by eric1tran — APPROVED** [review]

File: public_html/layers.js [hunk 1]
```diff
@@ -113,30 +113,27 @@ function createBaseLayers() {
...
 
-                for (var type in chartbundleTypes) {
+                faa_tiles.forEach(tile => {
                         us.push(new ol.layer.Tile({
-                                source: new ol.source.TileWMS({
-                                        url: 'http://wms.chartbundle.com/wms',
-                                        params: {LAYERS: type},
-                                        projection: 'EPSG:3857',
-                                        attributions: 'Tiles courtesy of <a href="http://www.chartbundle.com/">ChartBundle</a>'
+                                source: new ol.source.XYZ({
```
Comment (mutability) [inline public_html/layers.js:125]: Should we use a TileArcGISRest layer here, not XYZ?

https://openlayers.org/en/latest/examples/arcgis-tiled.html

File: public_html/layers.js [hunk 2]
```diff
@@ -113,30 +113,27 @@ function createBaseLayers() {
...
+                                    url: `https://tiles.arcgis.com/tiles/ssFJjBXIUyZDrSYZ/arcgis/rest/services/${tile.name}/MapServer/tile/{z}/{y}/{x}`,
+                                    attributions: 'Tiles courtesy of <a href="http://tiles.arcgis.com/">arcgis.com</a>',
```
Comment (mutability) [inline public_html/layers.js:127]: Probably should reference the FAA, or the FAA arcgis page, here. arcgis is the platform, not the ultimate data source.

File: public_html/layers.js [hunk 3]
```diff
@@ -113,30 +113,27 @@ function createBaseLayers() {
...
+                                    transition: 0,
                                 }),
-                                name: 'chartbundle_' + type,
-                                title: chartbundleTypes[type],
-                                type: 'base',
-                                group: 'chartbundle'}));
-                }
+                                name: tile.name,
+                                title: tile.title,
+                                type: 'base'
```
Comment (mutability) [inline public_html/layers.js:134]: Possibly we need minZoom/maxZoom on the TileLayer too?
Setting `extent` may also help avoid trying to load tiles that do not exist.

**Review by mutability — CHANGES_REQUESTED** [review]

File: public_html/layers.js (hunk 1 above)
Comment (nikeflight) [reply public_html/layers.js:141]: I've tried the TileArcGISRest endpoint but for whatever reason, the server tells me the functionality is not supported.

GitHub won't let me upload an image for some reason.

File: public_html/layers.js (hunk 2 above)
Comment (nikeflight) [reply public_html/layers.js]: Changed.

File: public_html/layers.js (hunk 3 above)
Comment (nikeflight) [reply public_html/layers.js]: I've added different zoom levels for each map.

I've also added a rough extent for each map EXCEPT IFR_HIGH.
IFR_HIGH is really big and extends across the Pacific to Manila. So I think putting an extent for that is pointless.

**Pull request merged**